library/basics/hedberg.red

```diff
@@ -46,3 +47,27 @@ def paths-stable→set (A : type) (st : (x y : A) → stable (path A x y)) : is-
 -- Hedberg's theorem for decidable path types
 def discrete→set (A : type) (d : discrete A) : is-set A =
   paths-stable→set A (λ x y → dec→stable (path A x y) (d x y))
+
+def mere-relation/set-equiv 
```

Comment:
@mortberg I'm not a fan of the terminology `mere-relation`; what is preferred in your Thought?

Reply:
I never liked the word "mere" for hProp valued things... Why not just call it `relation` or `rel`?

Reply:
For me simply "relation" is anything of type `A → B → type`. In [quotient.red](https://github.com/RedPRL/redtt/blob/master/library/cool/quotient.red#L61) a propositional relation is called `prop-valued`, is that good enough?

Reply:
Hmm, what about `hrel` or `hrelation`? (just like we use `hProp` for the universe of h-propositions)

Reply:
@jonsterling any preference? :)

Reply:
`hrel` is the name that Voevodsky used: https://github.com/UniMath/UniMath/blob/master/UniMath/Foundations/Sets.v#L474

I would go for that

Reply:
@mortberg Sounds good to me!